nlg/README.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Natural Language Generation
+prefix: Natural Language Generation
+...
+
+[TOC]
+
+From v1.54, Gramex includes a Natural Language Generation component. The NLG app
+is a smart template creator which:
+
+1. Automatically creates tornado templates from English text in the context of a dataset.
+2. Allows for modification and generalization of these templates.
+3. Renders these templates as a unified narrative.
+
+To use it, add the following to your `gramex.yaml`:
+
+```yaml
+import:
+  nlg:
+    path: $GRAMEXAPPS/nlg/gramex.yaml
+    YAMLURL: $YAMLURL/nlg/
+    auth:
+      login_url: /$YAMLURL/login
+```
+
+This configuration mounts the app at [nlg/](nlg/):
+
+<div class="example">
+  <a class="example-demo" href="nlg/">NLG</a>
+  <a class="example-src" href="http://github.com/gramener/gramex/blob/master/gramex/apps/guide/nlg/gramex.yaml">Source</a>
```

Comment:
Could you change the link to  http://github.com/gramexrecipes/gramex-guide/blob/master/nlg/gramex.yaml ?

Reply:
Done.